site/content/docs/5.2/getting-started/parcel.md

```diff
@@ -1,101 +1,149 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-  </head>
-  <body>
-    <script type="module" src="./index.js"></script>
-  </body>
-</html>
-```
+   ```json
+   {
+      // ...
+      "scripts": {
+        "start": "parcel serve src/index.html --public-url / --dist-dir dist",
+        "test": "echo \"Error: no test specified\" && exit 1"
+      },
+      // ...
+   }
+   ```
 
-### Edit `package.json`
+3. **Fill in the `src/index.html` file.** Parcel needs a page to render, so we use our `index.html` page to setup some basic HTML, including our CSS and JavaScript files.
 
-Add `dev` and `build` scripts in your `package.json` file.
+   ```html
+   <!doctype html>
+   <html lang="en">
+     <head>
+       <meta charset="utf-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1">
+       <title>Bootstrap w/ Parcel</title>
+       <link rel="stylesheet" href="scss/styles.scss">
+       <script type="module" src="js/main.js"></script>
+     </head>
+     <body>
+       <div class="container py-4 px-3 mx-auto">
+         <h1>Hello, Bootstrap and Parcel!</h1>
+         <button class="btn btn-primary">Primary button</button>
+       </div>
+     </body>
+   </html>
+   ```
 
-```json
-"scripts": {
-  "dev": "parcel ./src/index.html",
-  "prebuild": "npx rimraf build",
-  "build": "parcel build --public-url ./ ./src/index.html --dist-dir build"
-}
-```
+   Parcel can automatically detect we're using Sass and install the [Sass Parcel plugin](https://parceljs.org/languages/sass/) to support it. However, if you wish, you can also manually run `npm i --save-dev @parcel/transformer-sass`.
 
-### Run dev script
+4. **And finally, we can start Parcel.** From the `my-project` folder in your terminal, run that newly added npm script:
 
-Your app will be accessible at `http://127.0.0.1:1234`.
+   ```sh
+   npm start
+   ```
 
-```sh
-npm run dev
-```
+   <img class="img-fluid" src="/docs/{{< param docs_version >}}/assets/img/guides/parcel-dev-server.png" alt="Parcel dev server running">
 
-### Build app files
+At this point, everything is in the right place, but we don't have any styles or JavaScript from Bootstrap yet.
 
-Built files are in the `build/` folder.
+## Import Bootstrap
 
-```sh
-npm run build
-```
+Importing Bootstrap into Parcel requires two imports, one into our `styles.scss` and one into our `main.js`.
+
+1. **Import Bootstrap's CSS.** Add the following to `src/scss/styles.scss` to import all of Bootstrap's source Sass.
+
+   ```scss
+   @import "~bootstrap/scss/bootstrap";
+   ```
+
+   *You can also import our stylesheets individually if you want. [Read our Sass import docs]({{< docsref "/customize/sass#importing" >}}) for details.*
+
+2. **Import Bootstrap's JS.** Add the following to `src/js/main.js` to import all of Bootstrap's JS. Popper will be imported automatically through Bootstrap.
+
+   <!-- eslint-skip -->
+   ```js
+   import * as bootstrap from 'bootstrap'
```

Comment:
Not very important but there is an inconsistency between the non-use of ";" at the end of this JS line and the other block just after which uses it ` import Alert from 'bootstrap/js/dist/alert';`.
Same thing is observed in the Webpack guide.